File: Main Program/Python/Program Random/FizzBuzz.py
def fizzbuzz(n,x=1):
    if x>n:
        return 0
    if x % 3 == 0 and x % 5 == 0 :
        print ('fizz buzz')
        fizzbuzz(n,x+1)
    elif x % 3 ==0:
        print('fizz')
        fizzbuzz(n,x+1)
    elif x % 5 == 0:
        print('buzz')
        fizzbuzz(n,x+1)  
    else:
        print(x)
        fizzbuzz(n,x+1)  

fizzbuzz(20)

File: Main_Program/JAVA_OOP/src/test/FizzBuzz.java
package test;

public class FizzBuzz {
    public static void main(String[] args) {
        for (int x = 1;x <= 30;x++){
            if (x % 3 == 0 && x % 5 == 0){
                System.out.println("Fizz Buzz");
            } else if (x % 3 == 0){
                System.out.println("Fizz");
            } else if (x % 5 == 0){
                System.out.println("Buzz");
            } else {
                System.out.println(x);
            }
        }
    }
}
